### PR TITLE
Fix/replace fetch records

### PR DESCRIPTION
**Pull request by LiubomyrVV**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -360,4 +360,6 @@ MigrationBackup/
 .ionide/
 
 # Fody - auto-generated XML schema
-FodyWeavers.xsd
\ No newline at end of file
+FodyWeavers.xsd
```
Comment (KanasiOverflow) [inline .gitignore:363]: What's that?

File: Frontend/adminpanel/src/components/UI/button/GeneralButton.jsx [hunk 2]
```diff
@@ -8,5 +8,5 @@ const GeneralButton = React.memo(({ children, ...props }) => {
     </button>
   );
 });
-
```
Comment (KanasiOverflow) [inline Frontend/adminpanel/src/components/UI/button/GeneralButton.jsx:11]: Again, unnessecary changes

**Review by KanasiOverflow — CHANGES_REQUESTED** [review]
I'm waiting for changes

File: Frontend/adminpanel/src/context/ModalContext.jsx [hunk 3]
```diff
@@ -48,11 +58,12 @@ export const ModalProvider = ({ children }) => {
 
     const editRecord = useCallback(async (record) => {
         try {
+            console.log(record, currentTable)
```
Comment (KanasiOverflow) [inline Frontend/adminpanel/src/context/ModalContext.jsx:61]: This mustn't be in the repo!

File: Frontend/adminpanel/src/pages/RecordIdPage.jsx [hunk 4]
```diff
@@ -16,13 +27,36 @@ export default function RecordIdPage() {
...
+        openEditModal(record);
+    }, [openEditModal, record]);
+
+    const handleRemoveRecord = useCallback(() => {
+        removeRecord(record);
+    }, [removeRecord, record]);
+    
     useEffect(() => {
-        fetchRecordById(params.table, params.id);
+        console.log(fetchRecordById(params.table, params.id));
```
Comment (KanasiOverflow) [inline Frontend/adminpanel/src/pages/RecordIdPage.jsx:43]: This mustn't be in the repo!

File: Frontend/adminpanel/src/pages/RecordIdPage.jsx [hunk 5]
```diff
@@ -16,13 +27,36 @@ export default function RecordIdPage() {
...
+    const handleOpenEditModal = useCallback(() => {
+        openEditModal(record);
+    }, [openEditModal, record]);
+
+    const handleRemoveRecord = useCallback(() => {
+        removeRecord(record);
+    }, [removeRecord, record]);
+    
     useEffect(() => {
-        fetchRecordById(params.table, params.id);
```
Comment (KanasiOverflow) [inline Frontend/adminpanel/src/pages/RecordIdPage.jsx:20]: Why did you remove that?

File: Frontend/adminpanel/src/pages/RecordIdPage.jsx [hunk 6]
```diff
@@ -16,13 +27,36 @@ export default function RecordIdPage() {
...
+
+    const handleOpenEditModal = useCallback(() => {
+        openEditModal(record);
+    }, [openEditModal, record]);
+
+    const handleRemoveRecord = useCallback(() => {
+        removeRecord(record);
+    }, [removeRecord, record]);
+    
     useEffect(() => {
```
Comment (KanasiOverflow) [inline Frontend/adminpanel/src/pages/RecordIdPage.jsx:42]: If this hook dont need it, it should be removed

**Review by KanasiOverflow — CHANGES_REQUESTED** [review]

File: Frontend/adminpanel/src/pages/RecordIdPage.jsx (hunk 6 above)
Comment (LiubomyrVV) [reply Frontend/adminpanel/src/pages/RecordIdPage.jsx:42]: This part of the code is necessary

Comment (KanasiOverflow) [reply Frontend/adminpanel/src/pages/RecordIdPage.jsx:42]: For what? It's just console.log code

**Review by KanasiOverflow — CHANGES_REQUESTED** [review]

**Review by KanasiOverflow — APPROVED** [review]
Approve

**Pull request merged**